e2e/app/run.go

```diff
@@ -106,8 +106,10 @@ func Deploy(ctx context.Context, def Definition, cfg DeployConfig) (*pingpong.XD
 		return nil, err
 	}
 
-	if err := DeployGasApp(ctx, def); err != nil {
-		return nil, err
+	if def.Testnet.Network.IsEphemeral() {
```

Comment:
wouldn't this be always true based on this check: https://github.com/omni-network/omni/pull/2212/files#diff-10e0402871463c5990837013ef6adf90343f91c6ff9dadf051e5e737644f3944R53